tests/test_github.py

```diff
@@ -2084,7 +2216,6 @@ def test_local_branch_name_different_than_tracking_branch_name(self, mocker: Any
                 feature
```

Comment:
There are 3 occurrences left in this file:

```
tests/test_github.py:1981:        launch_command('discover', '--checked-out-since=1 day ago')
tests/test_github.py:1989:        launch_command('discover', '--checked-out-since=1 day ago')
tests/test_github.py:1994:        launch_command('discover', '--checked-out-since=1 day ago')
```

Probably to be replaced as well

Reply:
Yeah, first two instances can be replaced but the last one is used to check if the result of the executed commands is correct.